src/Console/AutomaticInstallation.php

```diff
@@ -100,12 +100,12 @@ protected function getHelp()
 HELP;
 	}
 
-	public function __construct(App\Mode $appMode, Cache $configCache, IManageConfigValues $config, Database $dba, array $argv = null)
+	public function __construct(App\Mode $appMode, Cache $keyValueCache, IManageConfigValues $config, Database $dba, array $argv = null)
 	{
 		parent::__construct($argv);
 
 		$this->appMode     = $appMode;
-		$this->configCache = $configCache;
+		$this->configCache = $keyValueCache;
```

Comment:
Is this meant to be changed?

Reply:
arg... yes you're right .. This was a accidentally executed PHPSTORM "auto-renaming" command, and I wondered what happened, now I know ..